service/switchboardapp/src/test/java/com/switchboard/app/DecoderControllerTest.java

```diff
@@ -0,0 +1,114 @@
+package com.switchboard.app;
+
+import com.switchboard.app.controller.DecoderController;
+import com.switchboard.app.dao.DecoderDaoImpl;
+import com.switchboard.app.dao.DeviceDaoImpl;
+import com.switchboard.app.domain.DecoderEntity;
+import com.switchboard.app.domain.DeviceEntity;
+import com.switchboard.app.exceptions.DeviceNotFoundException;
```

Comment:
`import com.switchboard.app.domain.DecoderEntity;` needs to change to `import com.switchboard.app.entity.DecoderEntity;`

`import com.switchboard.app.domain.DeviceEntity;` needs to change to `import com.switchboard.app.entity.DeviceEntity;`

`import com.switchboard.app.exceptions.DeviceNotFoundException;` needs to change to `com.switchboard.app.exceptions.ExceptionType;`

If not changed, we will get a compile error on merge.

Reply:
Has been updated